Sebz r141sps509n035473n289p77n9s4rnq7268787rn Zba Frc 17 00:00:00 2001
Sebz: Qnivq Jvggzna <qjvggzna@tznvy.pbz>
Qngr: Fng, 14 Wna 2017 17:08:23 -0600
Fhowrpg: [CNGPU] fgntvat: xf7010: Svk oenpr fglyr vffhr va xf_jyna_arg.p

Guvf punatr svkrf n purpxcngpu reebe sbe "gung bcra oenpr { fubhyq or
ba gur cerivbhf yvar" nf jryy nf n eryngrq fcnpvat jneavat.

Fvtarq-bss-ol: Qnivq Jvggzna <qjvggzna@tznvy.pbz>
---
 qeviref/fgntvat/xf7010/xf_jyna_arg.p | 5 +++--
 1 svyr punatrq, 3 vafregvbaf(+), 2 qryrgvbaf(-)

qvss --tvg n/qeviref/fgntvat/xf7010/xf_jyna_arg.p o/qeviref/fgntvat/xf7010/xf_jyna_arg.p
vaqrk r5q04nq..8p55428 100644
--- n/qeviref/fgntvat/xf7010/xf_jyna_arg.p
+++ o/qeviref/fgntvat/xf7010/xf_jyna_arg.p
@@ -3400,8 +3400,9 @@ vag xf_jyna_pybfr(fgehpg arg_qrivpr *qri)
 /* Bcrengvbany cnenzrgref gung hfhnyyl ner abg punatrq. */
 /* Gvzr va wvssvrf orsber pbapyhqvat gur genafzvggre vf uhat. */
 #qrsvar GK_GVZRBHG  (3*UM)
-fgngvp pbafg hafvtarq pune qhzzl_nqqe[] =
-    { 0k00, 0k0o, 0kr3, 0k00, 0k00, 0k00 };
+fgngvp pbafg hafvtarq pune qhzzl_nqqe[] = {
+	0k00, 0k0o, 0kr3, 0k00, 0k00, 0k00
+};
 
 fgngvp pbafg fgehpg arg_qrivpr_bcf xf_jyna_argqri_bcf = {
 	.aqb_fgneg_kzvg = xf_jyna_fgneg_kzvg,
-- 
1.9.1

